context/app/static/js/components/Detail/FileBrowserDUA/FileBrowserDUA.jsx

```diff
@@ -26,16 +26,20 @@ function FileBrowserDUA(props) {
       aria-describedby="alert-dialog-description"
     >
       <DialogContent>
-        <DialogContentText id="alert-dialog-description">
-          <h1 id="alert-dialog-title">
+        <StyledDiv id="alert-dialog-description">
+          <StyledHeader id="alert-dialog-title" variant="h2" component="h1">
             {'HuBMAP '} <ObliqueSpan>{`${title} Data`}</ObliqueSpan> {' Usage'}
-          </h1>
+          </StyledHeader>
 
-          <h2>Appropriate Use</h2>
-          <p>{appropriateUse}</p>
+          <StyledHeader variant="h4" component="h2">
```

Comment:
This has been confusing me... if `<h1>` and `<h2>` don't give us the control we need, could we define `<H1>` and `<H2>`? I don't know how we know which variant is used.

Reply:
Variant determines the styles used from the theme and the component (if a component prop isn't defined). Component overrides the variant component and is the element used in the DOM.

Reply:
`Typography` allows us to use the font styles defined in theme easily. This is just the MUI way of doing things, we can find an alternative if it doesn't suit us?